DfE.FindInformationAcademiesTrusts/Pages/Shared/_CookieBanner.cshtml

```diff
@@ -0,0 +1,44 @@
+@{
+  var returnPath = string.IsNullOrWhiteSpace(Context.Request.Query["returnPath"]) ? Context.Request.Path + Context.Request.QueryString : Context.Request.Query["returnPath"].ToString();
+}
+
+@if (!Context.Request.Cookies.ContainsKey(CookiesModel.ConsentCookieName) && TempData["DeleteCookies"] is null)
```

Comment:
This should be in a unit-testable method, perhaps alongside `OptionalCookiesAreAccepted`